inc/namespace.php

```diff
@@ -95,6 +99,27 @@ function add_pat_setting() {
 	] );
 }
 
+/**
+ * Register the translate setting and add the field.
+ */
+function add_translate_setting() {
+	add_settings_field(
+		'dc-translate',
+		__( 'Translate changelog to current language ?', 'js-dashboard-changelog' ),
+		__NAMESPACE__ . '\\render_translate_settings_field',
+		'general',
+		'default',
+		[
+			'label_for' => 'dc-translate'
+		]
+	);
+
+	register_setting( 'general', 'dc-translate', [
+		'sanitize_callback' => 'sanitize_text_field',
```

Comment:
Not sure if this is the right sanitization function to use for a boolean value (which is what we should be saving). It might require a simple custom sanitization function if one does not already exist.